packages/sdk/src/schema/resources.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod";
+
+const ResourceId = z.string();
+
+export const Resource = z.union([
+  // !!! this is prototype only resource type
+  // build other resources before exposing to production
+  z.object({
+    type: z.literal("getjson"),
```

Comment:
probably need a better type name, so if I see correctly you want different types for get, post etc and the type also is specific for a json response? 

So if you wanted to fetch text, you would create "gettext"?  as a ressource type? Almost feels like method and responseType should be separate fields there

Reply:
No, it's not about methods. It's just a configuration type. Could be grapnel, could be full blown rest api or just simple rest request.

Here type is for prototype only. We will come up with better resources when start integrating stuff.